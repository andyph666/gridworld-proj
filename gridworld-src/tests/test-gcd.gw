int a = 3333;
int b = 27;
print (b);
print(a+b);
print (10%3);

print (a);
print (b);
while(b>0){
	c = a % b;
	a = b;
	b = c;
}
print(a);