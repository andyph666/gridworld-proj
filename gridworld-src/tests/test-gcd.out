27
3360
1
3333
27
3
